translations/ui-orders/en.json

```diff
@@ -921,5 +922,6 @@
   "permission.settings.order-templates.delete": "Settings (Orders): Can view, edit, delete Order Templates",
   "permission.settings.order-templates.edit": "Settings (Orders): Can view, edit Order Templates",
   "permission.settings.order-templates.view": "Settings (Orders): Can view Order Templates",
-  "permission.settings.view": "Settings (Orders): Can view all settings"
+  "permission.settings.view": "Settings (Orders): Can view all settings",
+  "permission.settings.errors.permissionsRequired": "User does not have required permissions."
```

Comment:
Let's keep translations, that don't relate to defining translations for the permission sets out of this place.

`permission.*` - uses for perms from `package,json`

Reply:
updated